src/test/java/software/amazon/awssdk/crt/test/S3ClientTest.java

```diff
@@ -1194,67 +1197,6 @@ public void onFinished(S3FinishedResponseContext context) {
         }
     }
 
-    @Test
-    public void testS3GetChecksums() {
```

Comment:
it just repeats the same thing as the getting part of `testS3PutTrailerChecksums`, so, there is no reason to repeat it twice